themes/custom/az_barrio/css/style.css

```
@@ -6,11 +6,20 @@
  * of these @include files will be combined into a single file.
  */
 
+/* Class for Natural Image block alignment issue */
+.align-natural-image {
+  display:flex;
+}
+
 /* For structuring header utilities*/
 header#header_arizona {
   min-height: 10px;
 }
 
+.align-natural-image {
+  display:flex;
+}
+
```

Should these styles get removed now that you're using the `d-flex` class?

```suggestion
/* For structuring header utilities*/
header#header_arizona {
  min-height: 10px;
}
```